docs/zh/component_specs/sequencer/snowflake.md

```diff
@@ -0,0 +1,67 @@
+# Snowflake
```

Comment:
Please add this doc into the chinese sidebar `docs/zh/_sidebar.md`
![image](https://user-images.githubusercontent.com/26001097/194834138-f5534f99-8657-4557-834e-5b3e99e7c523.png)